Include link to documentation describing the script import process

SoryRawyer opened the pull request:
# Fixes #1606 

## Description

Includes a link to documentation describing the process to import a script during campaign creation.
I marked this with WIP since I'm not sure how to test this on mobile. I have tested this locally using Ubuntu with Firefox and Chrome. A screenshot from Chrome is included below the checklist.

Edit: I have also tested this on mobile (Pixel 2 with Chrome) and have attached a screenshot below the desktop screenshot.

# Checklist:

- [x] I have manually tested my changes on desktop and mobile
- [x] The test suite passes locally with my changes
- [x] If my change is a UI change, I have attached a screenshot to the description section of this pull request
- [x] [My change is 300 lines of code or less](https://github.com/MoveOnOrg/Spoke/blob/main/CONTRIBUTING.md#your-first-code-contribution), or has a documented reason in the description why it’s longer
- [x] I have made any necessary changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] My PR is labeled [WIP] if it is in progress

![Screenshot from 2020-07-03 22-52-19](https://user-images.githubusercontent.com/2566242/86503850-21dfcc80-bd80-11ea-8d0e-59f589683fe0.png)
![Screenshot_20200708-203850](https://user-images.githubusercontent.com/2566242/86983999-5cb87a80-c15b-11ea-993b-91b75c0f0aaa.png)

joemcl commented:
@SoryRawyer this points to the doc in /docs now I assume, and when we get /docs on GitHub page and/or readthedocs.org can update it right. I like the wiki idea but not sure that creates a directory structure of /docs and copies over the contents of docs automatically or updates to those.

joemcl commented:
OK right seeing it points to "https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO_IMPORT_GOOGLE_DOCS_SCRIPTS_TO_IMPORT.md"; .

ibrand commented on src/containers/AdminScriptImport.jsx:
```diff
@@ -52,11 +52,22 @@ export default class AdminScriptImport extends Component {
+          subtitle={
+            <span>
+              You can import interactions and canned responses from a properly
+              formatted Google Doc. Please refer to{" "}
```
What are the `{" "}`s surrounding the url for?

schuyler1d replied:
it forces a space between `to` and the `<a href>` link -- jsx/prettier is a bit annoying here, but this is the right way to keep the space.

SoryRawyer commented:
My apologies for the delay! I've added a mobile screenshot and removed the `WIP` from the PR title. Please let me know if there's anything more I should include on this PR!
